.github/workflows/Tests.yml

```diff
@@ -3,26 +3,27 @@ name: Tests
 on: 
   push:
   pull_request:
-    types: [opened]
+    types: [ opened, synchronize, reopened ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
I think concurrency doesn't work on m1 runners.

Reply:
Tested it out, following builds got canceled automatically (pushed a temporary commit & dropped it right after):
- https://github.com/poviolabs/PovioKit/actions/runs/7791423664
- https://github.com/poviolabs/PovioKit/actions/runs/7791411933

Superseded by: https://github.com/poviolabs/PovioKit/actions/runs/7791435474